scvi/model/base/_pyromixin.py

```diff
@@ -123,7 +126,7 @@ def train(
                 batch_size=batch_size,
                 use_gpu=use_gpu,
             )
-        training_plan = PyroTrainingPlan(pyro_module=self.module, **plan_kwargs)
+        training_plan = training_plan(pyro_module=self.module, **plan_kwargs)
```

Comment:
```suggestion
        training_plan = training_plan(self.module, **plan_kwargs)
```

making this general so that it wouldn't raise an error if someone used different kwarg names